### DISCUSSION TITLE
Does ente support subfolders in folders ?

### DISCUSSION THREAD

**Discussion by mckeyzhang** [ente-io/ente#1667] [Enhancements]
first of all, thank you all to make ente. i love it. it's very clear and useful.
but i'm puzzled whether it support subfolders or not. when i upload folders, it just recursive all the photos in it  and put them all in a same level. what i need is that it keeps the subfolders relationship when  A  >  B > C > D ... , the > means: including.
A-------img01
  --------img02
 --------img03
 --------B 
               -------imgB1
               -------imgB2
 --------C 
               -------imgC1
               -------imgC2
.......

by the way, it's better to keep some desciptions and names of the image. 

thanks a lot~

**Comment by vishnukvmd** [comment]
Hey, we don't support sub folders right now, but please join this discussion if you're interested: https://github.com/ente-io/ente/discussions/552 :)

I'll close this ticket in favor of that, but feel free to reopen if I missed something.
